common/xrdp_client_info.h

```diff
@@ -38,6 +46,11 @@ struct xrdp_client_info
   int bpp;
   int width;
   int height;
+  int physical_width;
+  int physical_height;
+  int orientation;
+  int desktop_scale_factor;
+  int device_scale_factor;
```

Comment:
These new variables must be appended to the end.

Reply:
fixed